features/comments_and_kudos/comments_adminposts.feature

```diff
@@ -112,3 +112,28 @@ Feature: Commenting on admin posts
       And I press "Comment"
     Then I should see "Comment created!"
       And I should see "zug zug"
+
+  Scenario: Modifying the comment permissions of an admin post with translations
+    Given I have posted an admin post
+      And basic languages
+      And I am logged in as a "translation" admin
+    When I make a translation of an admin post
+      And I follow "Back to AO3 News Index"
+      And I follow "Edit"
+      And I choose "Only registered users can comment"
+      And I press "Post"
+    Then I should see "Sorry, this news post doesn't allow non-Archive users to comment."
+    When I follow "Deutsch"
+    Then I should see "Sorry, this news post doesn't allow non-Archive users to comment."
+
+  Scenario: Translation of admin post with comments disabled
+    Given I have posted an admin post with comments disabled
+      And basic languages
+      And I am logged in as a "translation" admin
+    When I make a translation of an admin post
+    Then I should see "Sorry, this news post doesn't allow comments."
+    When I follow "Edit Post"
+    Then I should see "No one can comment"
+      And I should not see "Registered users and guests can comment"
```

Comment:
I don't like these negative assertions because they won't fail if we change the text of the options. Can we do without them?

(A much better test would be to check for a selected disabled radio button "No one can comment" option, but as discussed elsewhere we're not doing that this issue.)

Reply:
Hmm. I wanted something to test that the section with the radio buttons was missing when you went to edit the post, since otherwise that's totally uncovered. Maybe I can check any make sure the page has no radio buttons? I don't think there are any others on that page.

Reply:
We'll need a new test step for that as well. I can live with a TODO.